index.html

```diff
@@ -859,13 +858,26 @@ <h3>
           application, it SHOULD treat that manifest as a description of a
           distinct application, even if it is served from the same URL as that
           of another application. When the user agent sees a manifest where
-          |manifest|["id"] is [=url/equal=] with [=URL serializer/exclude
-          fragment|exclude fragment true=] to the [=identity=] of an
-          already-installed application, it SHOULD be used as a signal that
+          |manifest|["id"] is [=url/equal=] (OPTIONAL: with
+          [=URL/equals/exclude fragments=] set to true) to the [=identity=] of
+          an already-installed application, it SHOULD be used as a signal that
           this manifest is a replacement for the already-installed
           application's manifest, and not a distinct application, even if it is
           served from a different URL than the one seen previously.
         </p>
+        <p class="note" title="Exclude fragments for good measure">
```

Comment:
```suggestion
        <p class="note" title="Exclude fragments for comparison">
```

Reply:
Rephrased as "is best practice". The "good measure" was supposed to convey the fact that you don't really need to do it, but if you can then just do it anyway to sanitize bad data.

I hope this also conveys that user agents _don't_ have to do this and it can be beneficial not to, because it affords optimizations like hashing.